src/controllers/hashtags.controller.js

```diff
@@ -0,0 +1,14 @@
+import { serverErrorResponse, okResponse } from "./controllers.helper.js";
+
+import { getTrends } from "../repositories/hashtags.repository.js";
+
+async function GetTrendingHashtags(req, res) {
```

Comment:
Sugestão: Vamos seguir o padrão do back, que é fazer com que as funções comecem com letra miniscula (:

Reply:
Nossa nem tinha visto, força do hábito, eu sempre faço com maiusculas e devo ter feito sem perceber. Valeu!